src/index.js

```diff
@@ -1 +1,39 @@
+import {
+  showProjectList,
+  showProjectForm,
+  createProjectsArray,
+  createDefaultProject,
+}
+from "./controllers/helpers/projects/projectHelpers";
 
+import {
+  submitProjectForm
+} from "./controllers/helpers/projects/formHelpers";
+
+import {
+  submitTodoCallBack,
+} from './controllers/helpers/todos/todoHelpers';
+
+createProjectsArray();
+createDefaultProject();
+showProjectList();
+
+// For refactor:
+// Adding event listener to Project Form
+document.getElementById("projectNameForm").addEventListener('submit', e => {
```

Comment:
The `index.js` is probably where all the magic for ToDo List App happens. These DOM listeners need to be in their own file, like you did for the task and project helpers

Reply:
Fixed